.github/workflows/storefront-eslint.yml

```diff
@@ -0,0 +1,44 @@
+name: ESLint
+on:
+  workflow_call:
+    inputs:
+        extensionName:
+          description: 'Your extension name'
+          required: true
+          type: string
+
+jobs:
+  run:
+    name: ESLint
+    runs-on: ubuntu-latest
+    steps:
+      - name: Clone Shopware
+        uses: actions/checkout@v4
+        with:
+         repository: shopware/shopware
+         ref: trunk
+
+      - name: Clone Extension
+        uses: actions/checkout@v4
+        with:
+          path: custom/plugins/${{ inputs.extensionName }}
+
+      - name: Setup Node
+        uses: actions/setup-node@v4
+        with:
+          node-version: '20.x'
+
+      - name: Install Shopware Dependencies
+        working-directory: src/Storefront/Resources/app/storefront
+        run: npm install --no-audit --no-fund --prefer-offline
+
+      - name: Install Plugin Dependencies
+        working-directory: custom/plugins/${{ inputs.extensionName }}/src/Resources/app/storefront
+        run: npm install --no-audit --no-fund --prefer-offline
+
+      - name: Run ESLint
+        working-directory: custom/plugins/${{ inputs.extensionName }}/src/Resources/app/storefront
+        run: |
+          ../../../../../../../src/Storefront/Resources/app/storefront/node_modules/.bin/eslint --config ../../../../../../../src/Storefront/Resources/app/storefront/.eslintrc.js ./src
```

Comment:
```suggestion
          ${{ github.workspace }}/src/Storefront/Resources/app/storefront/node_modules/.bin/eslint --config ${{ github.workspace }}src/Storefront/Resources/app/storefront/.eslintrc.js ./src
```

Reply:
maybe plugins should bring their own config 😅

Reply:
Maybe we could also switch to https://biomejs.dev 🤔

Reply:
nice - maybe we can suggest it